Source/WebInspectorUI/UserInterface/Views/DOMNodeDetailsSidebarPanel.js

```diff
@@ -728,6 +737,7 @@ WI.DOMNodeDetailsSidebarPanel = class DOMNodeDetailsSidebarPanel extends WI.DOMD
                 this._accessibilityNodeInvalidRow.value = invalid;
                 this._accessibilityNodeLabelRow.value = label;
                 this._accessibilityNodeLiveRegionStatusRow.value = liveRegionStatusNode || liveRegionStatus;
+                this._accessibilityNodeSwitchStateRow.value = switchState;
```

Comment:
this will be hidden when it has no value (i.e. `""`) right?  if not, we should make sure this is only shown when it has a value

Reply:
Correct, that's the intention! It behaves this way with the PR as-is (the "State" row shows for switch controls, and no other type of element, as they have `switchState === ""`), but please let me know if there's something else that needs to be done or I've otherwise missed something.